### PR TITLE
Prep for ostree_sysroot_new_at()

### PR DESCRIPTION
**Pull request by cgwalters**

### CODE REVIEW

**Comment by cgwalters** [comment]
Hah, glad we actually had unit tests for the admin locking; my devshell didn't have `parallel` installed.

File: src/ostree/ot-main.c [hunk 1]
```diff
@@ -396,10 +396,12 @@ ostree_admin_option_context_parse (GOptionContext *context,
...
     {
-      GFile *path = ostree_sysroot_get_path (sysroot);
+      OstreeDeployment *booted = ostree_sysroot_get_booted_deployment (sysroot);
 
-      /* If sysroot path is "/" then user must be root. */
-      if (!g_file_has_parent (path, NULL) && getuid () != 0)
+      /* Only require root if we're manipulating a booted sysroot. (Mostly
+       * useful for the test suite)
+       */
+      if (booted)
```
Comment (jlebon) [inline src/ostree/ot-main.c]: This is missing a ` && getuid () != 0`, no?

Comment (cgwalters) [reply src/ostree/ot-main.c]: Yeah, see fixup ⬆️

Comment (jlebon) [reply src/ostree/ot-main.c]: Bah, my page hadn't refreshed. ♻️

**Comment by jlebon** [comment]
@rh-atomic-bot r+ add1835

**Pull request closed without merging**